1-exercises/F-array-forEach/exercise.js

```diff
@@ -9,6 +9,18 @@
 
 let arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
 
+arr.forEach((number) => {
+  if (number % 3 === 0 && number % 5 === 0) {
+    console.log("FizzBuzz");
+  } else if (number % 5 === 0) {
+    console.log("Buzz");
+  } else if (number % 3 === 0) {
+    console.log("Fizz");
+  } else {
+    console.log(number);
+  }
+});
```

Comment:
Yes, this works well. Another option would be to build up a string, so that you can concatenate "Fizz" and "Buzz" and not have to do the first check - but your method is equally good.